security/rage/Portfile

```diff
@@ -0,0 +1,440 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           cargo 1.0
+
+github.setup        str4d rage 0.10.0 v
+github.tarball_from archive
+revision            0
+categories          security
+installs_libs       no
+license             MIT Apache-2
+maintainers         {turbocool3r @turbocool3r} \
```

Comment:
An entry for a maintainer usually includes both the obfuscated email address, as well as Github handle.

So if your email was foobar@gmail.com, it would look like:

```
maintainers         {gmail.com:foobar @turbocool3r} \
```

Having just your email handle with no domain specified defaults to a macports.org email address.

Reply:
Thanks, fixed it.